examples/plugins/src/file-size.plugin.integration.test.ts

```diff
@@ -0,0 +1,117 @@
+import { vol } from 'memfs';
+import { beforeEach, describe, expect, it, vi } from 'vitest';
+import { executePlugin } from '@code-pushup/core';
+import { categoryRefSchema, pluginConfigSchema } from '@code-pushup/models';
+import { MEMFS_VOLUME } from '@code-pushup/testing-utils';
+import {
+  PluginOptions,
+  create,
+  audits as fileSizeAudits,
+  recommendedRefs,
+  pluginSlug as slug,
+} from './file-size.plugin';
+
+// Mock file system API's
+vi.mock('fs', async () => {
+  const memfs: typeof import('memfs') = await vi.importActual('memfs');
+  return memfs.fs;
+});
+vi.mock('fs/promises', async () => {
+  const memfs: typeof import('memfs') = await vi.importActual('memfs');
+  return memfs.fs.promises;
+});
+
+const outputDir = MEMFS_VOLUME;
```

Comment:
```suggestion
const outputDir = '/test';
```
I think it is perfectly fine to use the value directly, even by removing the `outputDir` variable itself.

Reply:
as we have gitignore settings i thought it is useful to have that folder shared. otherwise it required changing `.gitignore` too for each different folder.

Reply:
> as we have gitignore settings i thought it is useful to have that folder shared. otherwise it required changing `.gitignore` too for each different folder.

Aren't you confusing it with `tmp` folder? That's the one we use in tests with the real file system and is in `.gitignore`. But this is a virtual memfs volume, and the name `/test` is just a convention.